whylog/front/whylog_factory.py

```diff
@@ -0,0 +1,17 @@
+from whylog.config import YamlConfigFactory
```

Comment:
Front shall not know whether YamlConfigFactory or DbConfigFactory or whatever is used. Loading configuration is a responsibility of Config layer.